pkg/kube/resources_test.go

```diff
@@ -11,6 +11,8 @@ func TestGetResourcesFromPath(t *testing.T) {
 
 	assert.Equal(t, nil, err, "Error should be nil")
 
+	assert.Equal(t, "Path", resources.SourceType, "Should have type Path")
+	assert.Equal(t, "./test_files/test_1", resources.SourceName, "Should have filename as name")
```

Comment:
Is there value in also testing that `CreationTime` attribute exists, or is that not super important in this context?

Reply:
Nevermind - the CreationTime is added via the `CreateResourceProviderFromAPI` (moving comment down)